lib/http/request.rb

```diff
@@ -97,9 +97,10 @@ def initialize(opts)
     end
 
     # Returns new Request with updated uri
-    def redirect(uri, verb = @verb)
+    def redirect(uri, verb = @verb, cookies_raw: nil)
```

Comment:
I would've liked to inline `redirect` since it's strange to me to ask an `HTTP::Request` to redirect itself, as opposed to having the higher level controller, `Redirector` do it.

It'd also avoid having to `cookies_raw: nil` just to protect against api change.